packages/react-core/src/components/Badge/Badge.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { css, getModifier } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/components/Badge/styles.css';
+
+export const propTypes = {
+  isRead: PropTypes.bool,
+  children: PropTypes.node,
+  className: PropTypes.string
+};
+
+export const defaultProps = {
+  isRead: false,
+  children: '',
+  className: ''
+};
+
+const Badge = ({ isRead, className, children, ...props }) => (
+  <span
+    {...props}
+    className={css(
+      styles.badge,
+      getModifier(styles, isRead ? 'read' : 'unread'),
```

Comment:
Not a big deal, but you can do `isRead ?styles.modifiers.read : styles.modifers.unread` here. Right now the Babel plugin won't check that the modifiers inside getModifier exist. Using the above will make sure if core removes read or unread the build will fail. getModifier is usful for passing through sizes or variants though. This works as is though.